package.json

```diff
@@ -33,12 +33,11 @@
     "@types/google-protobuf": "^3.2.4",
     "@types/mocha": "^2.2.40",
     "@types/node": "^7.0.5",
-    "@bazel/typescript": "^0.0.10",
     "babel": "^6.5.2",
     "chai": "^3.5.0",
     "mocha": "^3.2.0",
     "source-map-support": "^0.4.14",
-    "tslint": "^4.5.1",
+    "tslint": "^5.9.1",
```

Comment:
The old version would depend on an old version of tsutils, which is incompatible with the new Bazel typescript rules.